src/main/rx/rx.c

```diff
@@ -515,6 +515,19 @@ void parseRcChannels(const char *input, rxConfig_t *rxConfig)
     }
 }
 
+void updateRSSISbus(uint32_t currentTime)
+{
+    static uint32_t rssiUpdateAt = 0;
+
+    if ((int32_t)(currentTime - rssiUpdateAt) < 0) {
+        return;
+    }
+    rssiUpdateAt = currentTime + DELAY_5_HZ;
+
+    uint16_t currentRssiPct = fetchSbusFrameLossPctAndResetCounters();
+    rssi = (uint16_t)((constrain(currentRssiPct, 0, 100) / 100.0f) * 1023.0f);
```

Comment:
Floating point is expensive.  `constrain(currentRssiPct, 0, 100) * 1023 / 100;` should produce identical results here,  `(constrain(currentRssiPct, 0, 100) * 1023 + 50) / 100;` will even round the result